static/monorail-2.0-sb.yaml

```diff
@@ -0,0 +1,152 @@
+swagger: "2.0"
+info:
+  version: "0.0.1"
+  title: RackHD 2.0 Southbound
```

Comment:
Is the plan to stick with the 1.1 paradigm of keeping these routes split over applying roles and access rules?